Plots/ColormapComparisons/CB_Rain.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+CB_Rain.py
+================
+
+This script illustrates multiple color schemes for color maps which will allow for those
+impacted by color blindness to see visualizations. Using rainbow color schemes not only
+cause confusion for those impacted by color blindness, but is a poor choice in color scheme 
```

Comment:
```suggestion
 can cause confusion for those impacted by color blindness, but is a poor choice 
```
I think this sounds a bit better. "in color sheme" is implied and not needed if you're more focused on brevity.